bandpass/fft.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+import numpy.fft as fft
+import matplotlib
+import matplotlib.pyplot as plt
+import vertica_python
+
+# don't have a display on (this) Unix box
```

Comment:
What is this file for?  I don't see it being included by highpass.py.  Can we have a comment here?